content/en/serverless/installation/java.md

```diff
@@ -80,25 +103,35 @@ dependencies {
 You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
 
 1. [Install the Datadog Forwarder if you haven't][2].
-2. [Subscribe the Datadog Forwarder to your function's log groups][5].
+2. [Subscribe the Datadog Forwarder to your function's log groups][6].
+
+### Monitor Java Lambda function cold starts
+
+Cold starts occur when your serverless applications receive sudden increases in traffic, and can occur when the function was previously inactive or when it was receiving a relatively constant number of requests. Users may perceive cold starts as slow response times or lag. Datadog highly recommends you configure a monitor on Java Lambda function cold starts, and use Datadog Serverless Insights to [keep cold starts to a minimum][7].
+
+{{< img src="serverless/java-monitor-cold-starts.png" alt="Monitor Java Lambda Function Cold Starts"  style="width:100%;">}}
+
+To create a Datadog monitor on Java Lambda function cold starts, follow the monitor creation steps [here][8] with the following criteria:
+- Metric Name: `aws.lambda.enhanced.invocations`
+- From: `runtime:java*` and `cold_start:true`
+- Alert Grouping: Multi Alert, trigger a separate alert for each `function_arn`
 
 ### Unified service tagging
 
-Although it's optional, Datadog highly recommends tagging you serverless applications with the `env`, `service`, and `version` tags following the [unified service tagging documentation][6].
+Although it's optional, Datadog highly recommends tagging you serverless applications with the `env`, `service`, and `version` tags following the [unified service tagging documentation][9].
 
 ## Explore Datadog serverless monitoring
 
-After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless Homepage][7].
+After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless Homepage][10].
 
-## Monitor custom business logic
+### Monitor custom business logic
```

Comment:
Should this be a sub-heading under "Explore Datadog Serverless Monitoring", or at an even level?

Reply:
It's intentional, I'm actually surprised it wasn't already a sub-heading :)